episodes/python-template.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Generating a Repository using a Template"
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can you quickly set up a new Python repository that follows best practices?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain why you should use a template to start a new repository
+- Demonstrate how to create a repository using the [NLeSC Python Template]
+- Demonstrate how to use the upgrade feature of [`copier`] to extend your
+  repository with extra features
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Why use a template?
+
+No matter how (in)experienced a programmer you are, using a template when
+starting a new project is always a good idea. Templates let you easily make
+use of the latest and greatest in best practices, without having to reinvent
+the wheel for every project.
+
+For people new to software development, this means you can trust in the prior
+work of many other experienced people, and only have to focus on those parts
+that are particularly relevant to your project. For experienced developers it
+means you don't have to manually copy over the boilerplate code from a previous
+project. This means you don't have to replace all your old values manually and
+won't have a chance to accidentally forget some of them, while also keeping you
+easily up to date with the latest and greatest instead of remaining stuck at
+a previous point in time.
+
+In both cases, using a trusted template will save you time, worries and make
+sure your new projects are never outdated from the start.
```

Comment:
Rephrase:

As someone who is new to programming, employing a template can help you to setup your first projects and help you adhere to coding standards such that your software can be understood by others.
In general employing templates saves time and helps to prevent confusion when copy pasting from old repositories. E.g. parameters and specific configurations are easily overlooked and not being edited. 

In both cases ....